manubot/cite/curie/__init__.py

```diff
@@ -129,64 +134,33 @@ def _download_bioregistry() -> None:
             *(x.lower() for x in resource.pop("synonyms", [])),
         }
         # remove invalid prefixes as per https://github.com/manubot/manubot/pull/306#discussion_r744125504
-        resource["all_prefixes"] = sorted(
+        resource["synonyms"] = sorted(
             filter(valid_prefix_pattern.fullmatch, all_prefixes)
         )
```

Comment:
Noting that we're going to use the initial key here of `synonyms` to refer to the Manubot modified synonym list, which has been all lowercase, includes the primary prefix, and removes bad prefixes.